content/docs/components/serving/kfserving.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "KFServing"
+description = "Using KFServing for serving Models"
```

Comment:
s/Models/models

The Kubeflow docs standard is to use title case for page titles, and sentence case everywhere else. There's more info in the [style guide](https://www.kubeflow.org/docs/about/style-guide/#use-capital-letters-sparingly).